packages/tpetra/core/src/Tpetra_BlockMultiVector_decl.hpp

```diff
@@ -462,20 +455,20 @@ class BlockMultiVector :
   ///   MultiVector as modified in the space in which you modified
   ///   it, by calling the modify() method with the appropriate
   ///   template parameter.
-  template<class TargetMemorySpace>
-  void sync () {
-    mv_.template sync<typename TargetMemorySpace::memory_space> ();
-  }
+  // template<class TargetMemorySpace>
```

Comment:
These functions are commented out.  Since they were part of the public interface, they should, instead, be deprecated.